src/backend/src/services/risks.services.ts

```diff
@@ -0,0 +1,130 @@
+import { Role, User } from '@prisma/client';
+import { Risk } from 'shared';
+import prisma from '../prisma/prisma';
+import { throwAccessDeniedError, throwError, throwNotFoundError } from '../utils/response.utils';
+import { hasRiskPermissions, riskQueryArgs, riskTransformer } from '../utils/risks.utils';
+
+/**
+ * Gets all the risks for the given project
+ * @param projectId the project to get the risks for
+ * @returns a list of risks
+ * @throws if the given project doesn't exist
+ */
+export const getRisksForProject = async (projectId: number): Promise<Risk[]> => {
+  const requestedProject = await prisma.project.findUnique({ where: { projectId } });
+  if (!requestedProject) throwNotFoundError('Project', projectId);
+
+  const risks = await prisma.risk.findMany({ where: { projectId }, ...riskQueryArgs });
+
+  return risks.map(riskTransformer);
+};
+
+/**
+ * Creates a Risk in the database
+ * @param user the user creating the risk
+ * @param projectId the project to create the risk for
+ * @param detail the detail (description) of the risk
+ * @returns the id of the successfully created risk
+ * @throws if the user does not have access to create a risk
+ */
+export const createRisk = async (user: User, projectId: number, detail: string): Promise<string> => {
```

Comment:
Really like how you commented on param, return, and throws description for every risk endpoint method. 
This way, it will be much easier for every ppl to understand our endpoint methods for risk, as it shows the description and data type for every parameter in every endpoint method.